lens/util/store.go

```diff
@@ -92,5 +95,14 @@ func (pb *proxyingBlockstore) AllKeysChan(ctx context.Context) (<-chan cid.Cid,
 	return outChan, nil
 }
 
-func (pb *proxyingBlockstore) HashOnRead(enabled bool) {
+func (pb *ProxyingBlockstore) HashOnRead(enabled bool) {
```

Comment:
Did the blockstore interface change?

Reply:
No, but I wanted to return a concrete type that can be used as a blockstore but can also expose some metrics. So I exported this.